smoketest/scripts/cli/test_vpp.py

```diff
@@ -1393,6 +1394,25 @@ def test_17_vpp_nat(self):
         _, out = rc_cmd('sudo vppctl show nat44 summary')
         self.assertIn(f'max translations per thread: {sess_limit} fib 0', out)
 
+    def test_18_vpp_host_resources(self):
+        max_map_count = '100000'
+        hr_path = base_path + ['settings', 'host-resources']
+
+        # Check if max-map-count has default value
+        self.assertEqual(sysctl_read('vm.max_map_count'), '65530')
+
+        # Change max-map-count and check
+        self.cli_set(hr_path + ['max-map-count', max_map_count])
+        self.cli_commit()
+
+        self.assertEqual(sysctl_read('vm.max_map_count'), max_map_count)
+
+        # We cannot decrease max-map-count value so expect '100000' not '65530'
+        self.cli_delete(hr_path + ['max-map-count'])
+        self.cli_commit()
+
+        self.assertEqual(sysctl_read('vm.max_map_count'), max_map_count)
```

Comment:
If we delete the `max-map-count` option, the expected default value is `65530`  and not `100_000`